### ISSUE TITLE
Crash when left-click-dragging when crafting

### ISSUE THREAD

**Issue by Omgard** [Chuunkyz/BenchmarkII-1.18#27]
Title says it all. Tested on dedicated server and on multiple clients. Normal crafting table, vanilla items.

**Comment by Zarepheth** [comment]
Since updating to 0.5.1, I'm having a similar problem. However, I'm in single player.

Opened a vanilla crafting grid (vanilla workbench), tried adding vanilla sticks and Tech Reborn sapphires. Before going any further, the game shutdown, leaving me at the desktop.

I've added a few extra mods to my Curse modpack profile:
* LambsDynamicLights
* Crawl
* PreciseBlockPlacing (v1.0.8)
* And something for inventory sorting (after a modpack update removed such)

I've attached a crash log: [crash-2022-01-06_19.13.23-client.txt](https://github.com/Chuunkyz/BenchmarkII-1.18/files/7825638/crash-2022-01-06_19.13.23-client.txt)
